--- giflib-5.0.4/configure.ac.bero	2013-02-28 00:59:59.073078993 +0100
+++ giflib-5.0.4/configure.ac	2013-02-28 01:00:10.149940513 +0100
@@ -7,7 +7,7 @@ m4_ifdef([AM_SILENT_RULES], [AM_SILENT_R
 
 dnl Note: config.h is not used in the current build
 dnl We leave this in place only to suppress an error message at autogen time
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS([config.h])
 
 dnl Checks for programs.
 AC_PROG_LIBTOOL
